Lab 6

Koshruk opened the pull request:
lab 6

Koshruk committed:
added email form to footer

Koshruk committed:
added content to modal window(styling unfinished)

Koshruk committed:
some changes to modal window styling

Koshruk committed:
added placeholder to the form in footer

Koshruk committed:
fixed shadows

Koshruk committed:
skip

Koshruk committed:
rounded checked svg

Koshruk committed:
changed button styling

Koshruk committed:
skip

vozniuk-lntu reviewed (approved):
1. в модалці має бути одна форма і інпутами і кнопкою
2. для кнопок додай курсор пойнтер
3. заокруглення модалки
4. у футері теж кнопку у форму

vozniuk-lntu commented:
https://docs.google.com/document/d/1FzHnG28SZG4KW04a9jU_Ohts-CHcP3Tq/edit?usp=sharing&ouid=104518963218417128491&rtpof=true&sd=true

https://docs.google.com/document/d/1mAvRjcfd-E3kw37c7l-6YtCCPDqmX0xz/edit?usp=sharing&ouid=104518963218417128491&rtpof=true&sd=true

Koshruk committed:
rounden corners in modal window
added cursor: pointer to all buttons
grouped all inputs and buttons related to them into forms
